### DISCUSSION TITLE
docs: Multiple schemas, but not all schemas

### DISCUSSION THREAD

**Discussion by FStefanni** [mikro-orm/mikro-orm#5570] [General]
Hi,

I am trying to create some entities in multiple schemas, but not all.
To be precise, at the moment I need to put some tables in all schemas, except the "public" schema.
Is there a way to achieve this?

Thank you
regards

PS: if missing, this could be a nice to have feature

**Comment by B4nan** [comment]
That's a pretty broad question. Maybe you could use https://mikro-orm.io/docs/multiple-schemas#wildcard-schema.
